CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - [commands] Added infrastructure for parameter names in argument parsers, for future improving clarity
 - [commands] Updated exception handling in command tests for more precise error reporting
 - [spigot] Change EnchantGlow's key and name so it wouldn't collide with other plugins
+- [commands] Added parameter names support to `SimpleArgumentMapper` & `SpigotArgumentMapper`
```

Comment:
Change to a clearer explanation for the users, maybe something like "Allow specifying names for command arguments"